### ISSUE TITLE
Accessibility - All Browser || Screen reader focus is missing for the close(X) button

### ISSUE THREAD

**Issue by SavvasAkmQamruzzaman** [ng-select/ng-select#1942]
**Describe the bug**
Observed that screen reader focus is missing for the close(X) button. When selecting any item from the list it comes up with cancel(X) the elements.  The click works fine to eliminate selection but the screen reader not focusing on the close button.

**Expected behavior**
Screen reader focus should be available for the cancel button.

**Screenshots**
![Screen Shot 2021-07-29 at 8 48 03 AM](https://user-images.githubusercontent.com/79347321/127523715-811cc969-64fc-45a8-93a0-e33828d6bf39.png)

Any OS, Any browser, Any accessibility tools.
